traefik/values.yaml

```diff
@@ -935,6 +935,8 @@ hub:
   # -- Name of `Secret` with key 'token' set to a valid license token.
   # It enables API Gateway.
   token: ""
+  # -- Array of namespaces to watch. If left empty, Traefik watches all namespaces.
```

Comment:
I have two comments about this:

1. Current code is also including traefik ns by default in the list. 
2. When rbac.namespaced is set, it's possible to watch all namespaces nonetheless.

Reply:
It's like we have already done for RBACs on other providers (Ingress, CRD, Gateway) and I updated documentation accordingly. Even if it's namespaced, users may want to add all namespaces here and restrict it after some tests, so we need to keep the freedom of doing it.